apitools/org.eclipse.pde.api.tools.tests/src/org/eclipse/pde/api/tools/builder/tests/ApiBuilderTest.java

```diff
@@ -1154,7 +1154,7 @@ private static String toString(List<IProject> projects) throws Exception {
 			contents.append(System.lineSeparator());
 			contents.append("    is open: " + project.isOpen()); //$NON-NLS-1$
 			contents.append(System.lineSeparator());
-			if (project.hasNature(JavaCore.NATURE_ID) && project.isAccessible()) {
+			if (Util.isJavaProject(project) && project.isAccessible()) {
```

Comment:
Whats the difference between `Util` and `ProjectUtils`?

I'm not really a fan of Util classes as they are often a sign of bad API and tend to collect all kind of stuff and dependencies, but do we really need two of them?

Reply:
> Whats the difference between `Util` and `ProjectUtils`?
> 
Which `ProjectUtils` are you referring? `Util` is a utility class for API tools that already exists and does much more than just this.

Reply:
> Which ProjectUtils are you referring? 

https://github.com/eclipse-pde/eclipse.pde/pull/941/files#diff-42ce900b7490497be3f213a0f5930372f29db30ff877df84fbfc78245cd7b392

Reply:
> I'm not really a fan of Util classes as they are often a sign of bad API and tend to collect all kind of stuff and dependencies, but do we really need two of them?

I want to emphasize that all changes in this PR are internal and no API is involved at all.
And yes I tried to reduce the duplications with this PR. I kept that method since `PluginProject` is not an API calls and I didn't want to add yet another friend to its package. But in the end I already added more so another one doesn't matter.